x-pack/plugins/observability/public/embeddable/slo/overview/slo_overview.tsx

```diff
@@ -28,8 +27,6 @@ export function SloOverview({
   onRenderComplete,
   reloadSubject,
 }: EmbeddableSloProps) {
-  const containerRef = useRef<HTMLDivElement>(null);
```

Comment:
I left a relevant comment to the slo_overview_grid file as well. Here's the reference we need to remove there as well.